catalog/security/banner-marking/pom.xml

```diff
@@ -88,17 +88,17 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.91</minimum>
+                                            <minimum>0.89</minimum>
```

Comment:
They probably didn't have to, as the overloaded method is now being invoked from the old signature and that should leave the coverage paths identical; however, to be safe I checked the coverage reported on my box and then set it to three-one-hundredths lower. I had previously used a .01 or .02 buffer to account for Jacoco discrepancies. That's probably the difference.